File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 14/07/2026, 14:32:10
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        if not nums:
            return 0
        k=0
        for i in range(0,len(nums)):
            if(nums[i]!=val):
                nums[k]=nums[i]
                k+=1
        return k
        

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 14/07/2026, 14:32:15
class Solution {
    public int removeElement(int[] nums, int val) 
    {
        int index=0,k=0;
        for(int i=0;i<nums.length;i++)
        {
            boolean flag=false;
            if(nums[i]==val)
            {
                flag=true;
            }
            if(!flag)
            {
                nums[index]=nums[i];
                index++;
                k++;
            }
            
        }
        return k;
    }
}